.npmrc

```diff
@@ -1,6 +0,0 @@
-public-hoist-pattern[]=*
```

Comment:
We want pnpm to act like npm so that we can guarantee successful build with npm

Reply:
Okay. I'm not sure how this either makes pnpm act more like npm or makes npm act more like pnpm.

Pnpm discourages the use of `public-hoist-pattern` https://pnpm.io/npmrc#public-hoist-pattern, hence my suspicion:
> This setting is useful when dealing with some flawed pluggable tools that don't resolve dependencies properly.

Also, while not having two flavors of lockfile committed to this repo, I believe that there can be problems with various `npm` tooling if the lockfile doesn't exist at all.